Garyan refactoring

aryan16 opened the pull request:
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [X] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

istio-testing commented:
Hi @aryan16. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=istio%2Fistio.io).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

aryan16 commented:
@xulingqing could you please have a look in this pr.

Arhell reviewed (commented):
/ok-to-test

craigbox reviewed (changes requested):
`content/en/docs/reference/config/security/authorization-policy/index.html` is generated from https://github.com/istio/api/tree/master/security/v1beta1.  Changes to it here will be overwritten.  You'll have to make this change in the protos in the reference docs.

justinpettit commented:
Can you please not delete content in one commit and then add it back in a different commit?  It makes it hard to track the changes.  If content is moved, it will be easier to follow within a commit.

aryan16 commented:
@justinpettit sorry I couldn't understand your comment properly. We want to have one single commit for the change right ? Right now we are following that only, the other commit which you see had few issues. I forgot to squash both the commits.

ericvn commented:
> 2. Added a new section named **Examples** in references/configuration/security/Authorization Policy.

As noted above by Craig, this change will need to be done in the istio/api repo (https://github.com/istio/api/blob/master/security/v1beta1/authorization_policy.pb.html) and that will be pulled in (there is a daily job that runs that will pull the change). Note that the file is generated from https://github.com/istio/api/blob/master/security/v1beta1/authorization_policy.proto.

aryan16 commented:
> 
noted.

aryan16 commented:
I have moved some of the authorization examples to ops/configurations/security/security-policy-examples. I will add the tests once I get approval for this change.
cc : @myidpt

istio-testing commented:
@aryan16: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
gencheck_istio.io | ba518b57c41baa60fcdd7a8a5ea8a6667f5b484a | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/gencheck_istio.io/1479173306211897344) | true | `/test gencheck_istio.io`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
gencheck_istio.io | ba518b57c41baa60fcdd7a8a5ea8a6667f5b484a | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/gencheck_istio.io/1479173306211897344) | true | `/test gencheck_istio.io`
lint_istio.io | ba518b57c41baa60fcdd7a8a5ea8a6667f5b484a | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/lint_istio.io/1479173306186731520) | true | `/test lint_istio.io`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

justinpettit commented:
> @justinpettit sorry I couldn't understand your comment properly. We want to have one single commit for the change right ? Right now we are following that only, the other commit which you see had few issues. I forgot to squash both the commits.

I'm not saying that it should only be a single commit.  I'm requesting that if you move text, please remove it from one file and add it to the other file in the same commit.  If you are making changes that are logically separate (e.g., authn changes and authz changes), those should be in separate commits.

aryan16 commented:
> > @justinpettit sorry I couldn't understand your comment properly. We want to have one single commit for the change right ? Right now we are following that only, the other commit which you see had few issues. I forgot to squash both the commits.
> 
> I'm not saying that it should only be a single commit. I'm requesting that if you move text, please remove it from one file and add it to the other file in the same commit. If you are making changes that are logically separate (e.g., authn changes and authz changes), those should be in separate commits.
>
> Understood!! In the new commit there are changes in 2 files. I removed content from one file and added the same content in the other file.

aryan16 commented:
cc : @justinpettit

istio-testing commented:
@aryan16: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

istio-testing commented:
@aryan16: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620550954120122368) | true | `/test doc.test.profile_default`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620550954120122368) | true | `/test doc.test.profile_default`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620550954157871104) | true | `/test doc.test.profile_demo`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620550954120122368) | true | `/test doc.test.profile_default`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620550954157871104) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620550954199814144) | true | `/test doc.test.profile_none`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620550954120122368) | true | `/test doc.test.profile_default`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620550954157871104) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620550954199814144) | true | `/test doc.test.profile_none`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620550954245951488) | true | `/test doc.test.multicluster`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620550954120122368) | true | `/test doc.test.profile_default`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620550954199814144) | true | `/test doc.test.profile_none`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620550954245951488) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620804402216439808) | true | `/test doc.test.profile_demo`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620550954199814144) | true | `/test doc.test.profile_none`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620550954245951488) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620804402216439808) | true | `/test doc.test.profile_demo`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620804402178691072) | true | `/test doc.test.profile_default`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620550954199814144) | true | `/test doc.test.profile_none`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620804402216439808) | true | `/test doc.test.profile_demo`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620804402178691072) | true | `/test doc.test.profile_default`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620804402287742976) | true | `/test doc.test.multicluster`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620804402216439808) | true | `/test doc.test.profile_demo`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620804402178691072) | true | `/test doc.test.profile_default`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620804402287742976) | true | `/test doc.test.multicluster`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620804402254188544) | true | `/test doc.test.profile_none`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620804402216439808) | true | `/test doc.test.profile_demo`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620804402178691072) | true | `/test doc.test.profile_default`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620804402254188544) | true | `/test doc.test.profile_none`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620804402178691072) | true | `/test doc.test.profile_default`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620804402254188544) | true | `/test doc.test.profile_none`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620804402178691072) | true | `/test doc.test.profile_default`
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620853478744133632) | true | `/test doc.test.profile_default`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620853478744133632) | true | `/test doc.test.profile_default`
doc.test.profile_minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_minimal_istio.io/1621196761714397184) | true | `/test doc.test.profile_minimal`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620853478744133632) | true | `/test doc.test.profile_default`
doc.test.profile_minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_minimal_istio.io/1621196761714397184) | true | `/test doc.test.profile_minimal`
doc.test.profile-none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-none_istio.io/1669403492839591936) | true | `/test doc.test.profile-none`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620853478744133632) | true | `/test doc.test.profile_default`
doc.test.profile_minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_minimal_istio.io/1621196761714397184) | true | `/test doc.test.profile_minimal`
doc.test.profile-none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-none_istio.io/1669403492839591936) | true | `/test doc.test.profile-none`
doc.test.profile-minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-minimal_istio.io/1669403492927672320) | true | `/test doc.test.profile-minimal`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620853478744133632) | true | `/test doc.test.profile_default`
doc.test.profile_minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_minimal_istio.io/1621196761714397184) | true | `/test doc.test.profile_minimal`
doc.test.profile-none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-none_istio.io/1669403492839591936) | true | `/test doc.test.profile-none`
doc.test.profile-minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-minimal_istio.io/1669403492927672320) | true | `/test doc.test.profile-minimal`
doc.test.profile-default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-default_istio.io/1669403492751511552) | true | `/test doc.test.profile-default`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@aryan16: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
doc.test.multicluster_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.multicluster_istio.io/1620853478861574144) | true | `/test doc.test.multicluster`
doc.test.profile_demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_demo_istio.io/1620853478781882368) | true | `/test doc.test.profile_demo`
doc.test.profile_none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_none_istio.io/1620853478819631104) | true | `/test doc.test.profile_none`
doc.test.profile_default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_default_istio.io/1620853478744133632) | true | `/test doc.test.profile_default`
doc.test.profile_minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile_minimal_istio.io/1621196761714397184) | true | `/test doc.test.profile_minimal`
doc.test.profile-none_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-none_istio.io/1669403492839591936) | true | `/test doc.test.profile-none`
doc.test.profile-minimal_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-minimal_istio.io/1669403492927672320) | true | `/test doc.test.profile-minimal`
doc.test.profile-default_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-default_istio.io/1669403492751511552) | true | `/test doc.test.profile-default`
doc.test.profile-demo_istio.io | 80835f00a21e651acc1e921d8c861e4ee00b4bee | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/10572/doc.test.profile-demo_istio.io/1669403492801843200) | true | `/test doc.test.profile-demo`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

craigbox reviewed (commented):
Hey @aryan16 ; sorry this sat for so long.  I think it might be good to go after a rebase?
